core.md

```diff
@@ -57,23 +57,13 @@ You may [apply to join some teams](https://links.neoforged.net/apply), but some
 
 #### Steering Council
 
-The Steering Council is the overall management and oversight of the Project.
+The Steering Council helps to ensure good communication within teams by making sure significant work, PRs, rule changes, or decision are backed by a team's vote that is outlined in [/voting.md](/voting.md). The result of that team's vote will be then enforced, whether it be the Maintainer team, Moderator team, or any other. While any team member can put up votes themselves, the Steering Council acts as a dedicated monitoring position to watch for anything that could slip by without a vote.
 
-They “steer” the project to keep it aligned with the long-term vision, hence its secondary name of Vision Team.
+In other words, the members of the Steering Council does not give out orders or demands. They cannot tell people what to do. Instead, if something comes up and a majority of the Steering Council members believes it is significant, they can say that the thing needs a decision from the whole team and put up a vote for that team. This helps teams be aware of what others members on their team are doing and helps reduce people being blindsided by big changes. It also addresses the problem of a single member making sweeping changes under other member's noses.
 
-The Council has at all times an odd number of active members, such that stalemates are not possible.
+The Steering Council has at all times an odd number of active members, such that stalemates are not possible when deciding if something warrants a vote or not. The members of the Steering Council are decided by an annual internal vote to ensure fairness and cycling of council members over time. Details on the Steering Council voting process can be found at [/steering-council-voting.md](/steering-council-voting.md).
```

Comment:
This `steering-council-voting.md` document doesn't exist yet. 🤔

Reply:
It will once Maty's PR is merged. He took the existing voting.md doc and renamed it to steering-council-voting.md which my PR will be linking to

Reply:
Maty PR is merged now